src/compiler/main.cc

```diff
@@ -120,9 +120,11 @@ namespace verona::compiler
       // Add nested includes to the work list.
       for (auto& include : file->modules)
       {
-        auto directory = input_file.remove_filename();
-        directory += "/" + (std::string)*include;
-        work_list.push(directory);
+        auto new_input_file = input_file.remove_filename();
+        if (new_input_file.empty())
+          new_input_file = ".";
+        new_input_file += "/" + (std::string)*include;
```

Comment:
I would sooner keep the directory name, but change this line to 
```
auto new_input_file = directory + "/" + (std::string)*include;
```

My original was not clear, but I think your change is unclear in a different way.